roles/client/defaults/main.yml

```diff
@@ -1,3 +1,13 @@
 ---
 client:
   self_signed_cert: false
+  names:
+    - python-keystoneclient
+    - python-glanceclient
+    - python-novaclient
+    - python-neutronclient
+    - python-cinderclient
+    - python-heatclient
+    - python-ironicclient
+    - python-swiftclient
+
```

Comment:
yeah, for a while I thought I was going to have to use packaging so I added a bunch of tasks and more vars ... turned out I didn't end up needing to,  but it seemed to make sense to leave this bit in for readability of the tasks file.